Rephrase comment on BigInt operators (#29334)
mdn/content: files/en-us/web/javascript/reference/global_objects/bigint/index.md

### Operators

Most operators support BigInts, however most do not permit operands to be of mixed types — both operands must be BigInt or neither:

- [Arithmetic operators](/en-US/docs/Web/JavaScript/Reference/Operators#arithmetic_operators): `+`, `-`, `*`, `/`, `%`, `**`
- [Bitwise operators](/en-US/docs/Web/JavaScript/Reference/Operators#bitwise_shift_operators): `>>`, `<<`, `&`, `|`, `^`, `~`
- [Unary negation (`-`)](/en-US/docs/Web/JavaScript/Reference/Operators/Unary_negation)
- [Increment/decrement](/en-US/docs/Web/JavaScript/Reference/Operators#increment_and_decrement): `++`, `--`

> Mixed operand types are allowed by the boolean-returning operators:

Reviewer: ```suggestion
The boolean-returning operators allow mixing numbers and BigInts as operands: 
```

+ * - % **
```

However, the unary operator (`+`) cannot be supported on BigInt due to conflicting usage in asm.js, so it has been omitted [in order to not break asm.js](https://github.com/tc39/proposal-bigint/blob/master/ADVANCED.md#dont-break-asmjs).

> All of the [Bitwise operators](/en-US/docs/Web/JavaScript/Reference/Operators) are supported as well, except for `>>>` (zero-fill right shift), as every BigInt value is signed.

Reviewer: I suggest rewording this entire section, so it reads like:

```md
Most operators accept BigInt operands. Some of them involving number logic include:

- [Increment and decrement](/en-US/docs/Web/JavaScript/Reference/Operators#increment_and_decrement): `++`, `--`
- [Unary negation (`-`)](/en-US/docs/Web/JavaScript/Reference/Operators/Unary_negation)
- [Arithmetic operators](/en-US/docs/Web/JavaScript/Reference/Operators#arithmetic_operators): `+`, `-`, `*`, `/`, `%`, `**`
- [Relational operators](/en-US/docs/Web/JavaScript/Reference/Operators#relational_operators) and [Equality operators](/en-US/docs/Web/JavaScript/Reference/Operators#equality_operators): `>`, `<`, `>=`, `<=`, `==`, `!=`, `===`, `!==`
- Unary `-`
- [Bitwise operators](/en-US/docs/Web/JavaScript/Reference/Operators#bitwise_shift_operators): `>>`, `<<`, `&`, `|`, `^`, `~`

Others, such as [logical operators](/en-US/docs/Web/JavaScript/Reference/Operators#binary_logical_operators), only use the truthiness of operands.

There are the following caveats:

- The unary operator (`+`) cannot be supported on BigInt due to conflicting usage in asm.js, so it has been omitted [in order to not break asm.js](https://github.com/tc39/proposal-bigint/blob/master/ADVANCED.md#dont-break-asmjs).
- Unsigned right shift (`>>>`) is the only bitwise operator that's unsupported, as every BigInt value is signed.
- All binary operators must have numbers or BigInts on both sides — they cannot be mixed. The only exceptions are the equality and relational operators — you can compare numbers with BigInts.
- The `/` operator works as expected with whole numbers, but operations with a fractional result will be truncated without returning any fractional digits.
```

The exact wording is up to you.

Author: I've rewritten the entire section, I believe I've gotten everything you mentioned.
